packages/swapkit/swapkit-core/src/aggregator/contracts/index.ts

```diff
@@ -27,6 +28,7 @@ export enum AGG_CONTRACT_ADDRESS {
   PANGOLIN = '0x942c6dA485FD6cEf255853ef83a149d43A73F18a',
   AVAX_GENERIC = '0x7C38b8B2efF28511ECc14a621e263857Fb5771d3',
   AVAX_WOOFI = '0x5505BE604dFA8A1ad402A71f8A357fba47F9bf5a',
+  TRADER_JOE = '0x3b7DbdD635B99cEa39D3d95Dbd0217F05e55B212',
```

Comment:
Should add chain prefix if possible, we may integrate TJ on an other chain.

Reply:
good point